.gitignore

```diff
@@ -19,8 +19,8 @@
 # This is where the result of the go build goes
 /output/**
 /output
-/_output/**
-/_output
+/_output*/**
```

Comment:
Yeah I wasn't completely sure why there were two lines in the first place.  I'll update.